### DISCUSSION TITLE
2021-06-09 React Test 中 act() 有什么作用？ ~ TJ (Thaddeus Jiang)

### DISCUSSION THREAD

**Discussion by giscus** [ThaddeusJiang/thaddeusjiang.github.io#42] [Announcements]
# 2021-06-09 React Test 中 act() 有什么作用？ ~ TJ (Thaddeus Jiang)

可能 Frontend 门槛真的太低了。竟然有自称熟练掌握 React 的人回答不上来 React Test act() 的用途。

https://thaddeusjiang.com/2021-06-09-react-test-zhong-act-you-shi-me-zuo-yong.html

**Comment by xiaoyaozi1010** [comment]
> "可能 Frontend 门槛真的太低了"

有都少业务都不需要写 ut，也并不是每个人都会主动去学习用不到的技术，即便学了也未必都能记得住，作为 linux 内核开发人员也未必能完全了解 linux 内核的每一个角落，你能说 “linux 内核开发的门槛真的太低了”。你上来就说前端门槛可太低了，来臭显摆你作为面试官的优越感么？
所谓的术业有专功，每个工程师工作内容不同，面向的用户不同造就技术储备并不一完全一致；再者如今的大前端概念泛滥，前端技术爆炸，早已不是十几年前会点 css 和 jquery 就可以称为工程师的年代了，各种前端、伪前端、大前端方向的技术已经不是紧跟潮流就能学的过来的。对行业，对技术请有点敬畏之心吧。

**Reply by ThaddeusJiang** [reply]
😂 不要断章取义啊。

> 可能 Frontend 门槛真的太低了。竟然有自称熟练掌握 React 的人回答不上来 React Test act() 的用途。

前提是`自称熟练掌握 React` ，我并没有要求所有前端都会 act()。
